deps/tools/common.mk

```diff
@@ -171,8 +172,8 @@ $$(build_staging)/$2.tgz: $$(BUILDDIR)/$2/build-compiled
 	rm -rf $$(build_staging)/$2
 	mv $$@.tmp $$@
 
-$$(build_prefix)/manifest/$(strip $1): $$(build_staging)/$2.tgz | $(build_prefix)/manifest
-	mkdir -p $$(build_prefix)
+$$(build_prefix)/manifest/$(strip $1): $$(build_staging)/$2.tgz
+	mkdir -p $$(build_prefix)/manifest
```

Comment:
Seems unnecessary? We usually use the gmake extention to do mkdir, instead of each separate recipe being involved

Reply:
Didn't work for me for some reason. Is there something special I need to do?

Reply:
It appears to have been working for everyone else